hack/tools/metadata-version-validator/README.md

```diff
@@ -0,0 +1,24 @@
+# Metadata Version Validator
+
+## Purpose
+
+A small utility that given a version number it will check that a release series is defined in the passed in metadata.yaml file.
+
+This is primarily for use in release pipelines.
+
+## Usage
+
+First create the binary by:
+
+```bash
+cd hack/tools
+go build -tags=tools -o bin/metadata-version-validator sigs.k8s.io/cluster-api/hack/tools/metadata-version-validator
+```
+
+Then run the binary to check a version:
+
+```bash
+bin/metadata-version-validator --file ../../cluster-api-provider-aws/metadata.yaml --version v2.8.0
```

Comment:
```suggestion
bin/metadata-version-validator --file ../../metadata.yaml --version v1.11.0
```

Maybe it's better to reference a file in CAPI so the docs example is self-contained? That is, I got an error trying to follow this doc naively.

Reply:
Yeah i think you are right @nalum . I will change this.